lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -13,6 +14,9 @@ local SIZE = {
   width = 7,
   right_justify = true,
   display = function(entry)
+    if not entry.stat then
+      log.debug("" .. entry.Path:absolute() .. " appears to be an invalid symlink")
+      return '0.0' end
```

Comment:
I think this should be just a `0` since there is no division.

Reply:
Agreed, corrected